koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -566,8 +570,6 @@ def convert_csv_to_parquet(self, csv_filename: os.PathLike):  # noqa: C901
                     )
                 )
                 self.post_processor.finalize_post_processing()
-            if self.create_table and not self.trino_table_exists.get(self.report_type):
-                self.create_parquet_table(parquet_filepath)
```

Comment:
Why does gcp invoice month impact creating parquet tables here at all?

Reply:
same as above, basically moving outside that for loop.